release/archives/linux/data-prepper-jdk-x64.sh

```diff
@@ -8,27 +8,21 @@ if [[ $# -ne 2 ]]
   then
     echo
     echo "Error: Paths to pipeline and data-prepper configuration files are required. Example:"
-    echo "./data-prepper-tar-install.sh config/example-pipelines.yaml config/example-data-prepper-config.yaml"
+    echo "./bin/data-prepper config/example-pipelines.yaml config/example-data-prepper-config.yaml"
     echo
     exit 1
 fi
 
 PIPELINES_FILE_LOCATION=$1
 CONFIG_FILE_LOCATION=$2
 DATA_PREPPER_BIN=$(dirname "$(realpath "$0")")
-DATA_PREPPER_HOME="$DATA_PREPPER_BIN/.."
-EXECUTABLE_JAR=$(ls -1 "$DATA_PREPPER_HOME"/lib/*.jar 2>/dev/null)
+DATA_PREPPER_HOME=`realpath "$DATA_PREPPER_BIN/.."`
+DATA_PREPPER_CLASSPATH="$DATA_PREPPER_HOME/lib/*"
```

Comment:
nit: I expected the variable to contain something like `org.opensearch.dataprepper.DataPrepperExecute` not a literal file path. Although, in context on line 34, it's clear how the variable is used.

Reply:
This is now a Java classpath. So it contains the directory to all the jars (we no longer have only one data-prepper-core, but all the jars are in lib/). When running Data Prepper, the script provides Java with both the classpath directories and the classname of the class with the Java `main` method.